docs/04-operation-guides/operations/README.md

```diff
@@ -12,12 +12,8 @@ This is a complete list of Kyma operation guides:
 * [Uninstall Kyma or components](07-uninstall-kyma.md)
 * [Back up Kyma](10-backup-kyma.md)
 * [Pass an access token in a request header](ac-01-pass-access-token-in-request-header.md)
-* [Grafana Dashboards for Kyma Eventing](evnt-01-eventing-dashboards.md)
-* [Kyma Eventing Metrics](evnt-02-eventing-metrics.md)
```

Comment:
I assume the metrics itself should be still documented, only the dashboards will be moved outside